core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
+            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
+            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
+            if (bestPluginId != null) {
+                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
```

Comment:
Why is this `gradle-plugins` here. In the PR you reference in the description it is `gradle-plugin`.